### ISSUE TITLE
About Aggregation and Challenge Leader Board

### ISSUE THREAD

**Issue by re20100801** [Elody-07/AWR-Adaptive-Weighting-Regression#3]
Dear Auther:
      Thank you for share your code, I have several Question about your work.

       --First about Aggregation part code in 'feature_tool.py' line 53-54
              offset_ht =  (offset_ht * mask).view(batch_size, jt_num, -1) # (B, jt_num, F*F)
              offset_ht = F.softmax(offset_ht * 30, dim = -1) # (B, jt_num, F*F)
          In above code, a mask is used to remove the effect of invalid pixel. 
          But, after Softmax function, ZEROs in offset_ht  caused by mask still have weights. 
          Is that as you expect?

       --Second, you claim an outstanding result in your AAAI paper, expecially on HANDS2017. 
        Did you other post-process method like MODEL ENSEMBLE?

        --Third, the AWR result on LEADERBOARD of HANDS2019 and HANDS2017 decrease a lot recently. 
         Is that an accident?

        Thank you in advance~!

**Comment by Elody-07** [comment]
1. Since we have scaled offset_ht, after softmax function, zeros should be less affective to the final results.
2. We do not use any post-process method like model ensemble. We show single network results in the paper.
3. HANDS2019 and HANDS2017 shows result of latest submission.  We have submitted other methods in the leaderboard so currently the result is not AWR.
